CONFIGDOC.md

```diff
@@ -69,9 +69,9 @@ You can specify build options for your MATLAB build by first selecting **Build o
 
 MATLAB exits with exit code 0 if the specified tasks run without error. Otherwise, MATLAB terminates with a nonzero exit code, which causes the step to fail.
 
-Starting in R2024a, you can access the result of running a MATLAB build interactively in your Jenkins interface. After your build runs, the Jenkins build summary page displays the number of tasks that ran, failed, and were skipped. You can click the **MATLAB Build Results** link on the page to access the table of task results. The table provides information about each task that was part of the MATLAB build. Clicking a task name in the table displays relevant build log information on the **Console Output** page.
+Starting in R2024a, you can access the result of running a MATLAB build interactively in your Jenkins interface. After your build runs, the Jenkins build summary page displays the number of tasks that ran, failed, and were skipped. You can click the **MATLAB Build Results** link on the page to access the table of task results. The table provides information about each task that was part of the MATLAB build. For example, if a task was skipped, the table displays the reason that the task was skipped. Clicking a task name in the table displays relevant build log information on the **Console Output** page.

```

Comment:
Could we change "access the result" -> "browse the results"? Or just change "result" to "results"?

The skip message seems like a somewhat niche example if we're just trying to list examples of the type of information found in the table since it doesn't apply to every task. I would expect the user to care more about name, status, and duration. Is there a secondary reason that we're pointing out the skip reason specifically?

Reply:
What do you mean by 'Or just change "result" to "results"?'

Reply:
> What do you mean by 'Or just change "result" to "results"?'

We switch from referring to a single result in the first sentence to using plural results in the third sentence. I think that using the plural "results" everywhere feels a little bit smoother to read. And I just thought that "browse" was a slightly more descriptive verb than "access" here. I don't normally think of myself as "accessing" results, it's more of a "browsing results", "viewing results", "checking results", etc...

Reply:
> I added the skip example because the "skip reason" sounded like an important aspect of this minor release. However, I am happy to remove the added sentence.

I am fine if we don't want to keep it since the updated screenshot shows skip reason now.

> We switch from referring to a single result in the first sentence to using plural results in the third sentence. I think that using the plural "results" everywhere feels a little bit smoother to read.

Agree, I prefer using results everywhere for consistency

Reply:
> think that using the plural "results" everywhere feels a little bit smoother to read. And I just thought that "browse" was a slightly more descriptive verb than "access" here. I don't normally think of myself as "accessing" results, it's more of a "browsing results", "viewing results", "checking results", etc...

I changed result to results and removed the skip-related sentence. Regarding the best verb to use here, I will defer to the editor. Thanks Sam!